internal/audit/audit.go

```
@@ -0,0 +1,129 @@
+		return err
+	}
+
+	// O_APPEND on POSIX is atomic for writes <= PIPE_BUF; one JSON line is
+	// well under that, so concurrent `rapg run` invocations don't interleave.
+	// #nosec G304 -- path is fixed at ~/.rapg/sessions.jsonl, derived from
+	// os.UserHomeDir(); not influenced by external input.
+	f, err := os.OpenFile(path, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0o600)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
```

golangci-lint (2.12.1)

[error] Error return value of `f.Close` is not checked

(errcheck)